substrate/frame/revive/src/tests.rs

```diff
@@ -946,6 +946,21 @@ fn storage_max_value_limit() {
 	});
 }
 
+#[test]
+fn storage_clear() {
```

Comment:
Otherwise it might be confused with the normal storage clearing.

```suggestion
fn clear_storage_on_zero_value() {
```